include/envoy/ssl/context_manager.h

```diff
@@ -38,6 +39,12 @@ class ContextManager {
    * Iterate through all currently allocated contexts.
    */
   virtual void iterateContexts(std::function<void(const Context&)> callback) PURE;
+
+  /**
+   * Access the private key operations manager, which is part of SSL
```

Comment:
Again, I think I would prefer some naming that indicates this is specialized for BoringSSL only (or make this a top-level `union` of the two types).

Reply:
I agree that it is implicit and unambiguous, and other names would probably be longer and more cumbersome. Do you think we will maintain a BoringSSL only stance long term given the work that @bdecoste is doing?

Reply:
(maybe @bdecoste also wants to take a look at this discussion :D )

Reply:
Yes, the decision made at the EnvoyCon in December was that we're going to only support BoringSSL in-tree, and the approach that @bdecoste is taking (standalone extension vs compile-time alternative) is a direct result of that discussion.

cc @mattklein123 @lizan @bdecoste

Reply:
OK, then we can keep this naming convention, resolved, thanks.